### ISSUE TITLE
Cleartext passwords logged

### ISSUE THREAD

**Issue by FlorianKrammel** [linuxmuster/linuxmuster-webui7#236]
Plugin: lnm_users
File: views.py
Function: handle_api_users_password( )

The parameter "password" contains the cleartext password e. g. from global-admin an is shown in the logs (journalctl).

**Comment by kiarn** [comment]
Hello @FlorianKrammel ,

Thanks for the report, I can confirm. It's necessary to handle this with `pexpect`, but I'm afraid I will not have the time to solve this the next 2 weeks. I will show at it then.

Arnaud

**Comment by kiarn** [comment]
Hello,

I made some deep investigations, and there's 2 points to correct : 

 - Ajenti logs all `subprocess` commands in DEBUG mode. Fix is [prepared](https://github.com/ajenti/ajenti/commit/85a59bba19b46d0f8ad8ad39f1205caae5870c8e) to let the possibility to hide sensitive commands and will be available with the next release.
 - the second point is the fact that `sophomorix-passwd` is run with `sudo`, which logs all commands in `/var/log/auth.log`. I made a PR here to fix this : https://github.com/linuxmuster/linuxmuster-base7/pull/142

Arnaud
